frontends/web/src/components/TaskOwnerPageComponents/Advanced.js

```diff
@@ -43,35 +47,29 @@ const Advanced = (props) => {
                         <Row>
                           <Form.Label column>Task Configuration</Form.Label>
                           <Col sm="12">
-                            <Form.Text
-                              id="paramsHelpBlock"
-                              muted
-                              className="border-bottom"
-                            >
+                            <Form.Text id="paramsHelpBlock" muted>
                               <span style={{ color: "red" }}>BETA Notice</span>:
```

Comment:
This isn't part of the PR, but we can probably remove the `BETA Notice` and just make it `Note`: